### ISSUE TITLE
README: add instructions how to install clang-format

### ISSUE THREAD

**Issue by certik** [symengine/symengine#1587]
It doesn't seem to be in standard ubuntu nor Conda, unless I missed it.

**Comment by isuruf** [comment]
It's in conda as `clangdev`.

**Comment by certik** [comment]
Cool. We should document in README how to use it, so that people don't have to wait for Travis to give them the diff.

**Comment by isuruf** [comment]
Agree. (@isuruf-bot, also gives the diff, but I find it easier to have clang-format run as a git webhook so that it's run every time I commit)

**Comment by lkeegan** [comment]
Instructions for installing clang-format & setting up a git precommit hook are now in [CONTRIBUTING.md](https://github.com/symengine/symengine/blob/master/CONTRIBUTING.md#installing-clang-format)
